python_training/tests/test_midas.py

```diff
@@ -0,0 +1,151 @@
+import pytest
+from python_training.base import Base
+from sqlalchemy import create_engine
+from python_training.member import add_member, Member
+from python_training.event import add_event, Event
+from python_training.organization import add_organization, Organization
+from python_training.utils import get_session
+from python_training.config import MEMBERS_INFO, ORGANIZATIONS_INFO, EVENTS_INFO
+from python_training.queries import members_that_are_not_at_organization_location, last_event_per_member, \
+    number_of_organizations_each_event, number_of_members_in_organization, people_you_may_know
+
+
+TEST_DB_FILE_NAME = 'midas_test.db'
+
+
+@pytest.fixture(scope='function')
+def test_db(tmpdir_factory):
+    db_file_path = tmpdir_factory.mktemp('db_dir').join(TEST_DB_FILE_NAME)
```

Comment:
`tmpdir_factory` is a bit of an overkill here, since you don't care about the temp dir's name.
Just use `tmpdir` instead.